visma/functions/constant.py

```diff
@@ -33,13 +39,192 @@ def differentiate(self):
         self.value = 0
 
     def integrate(self, intwrt):
-        self.coefficient = (self.value)**(self.power)
+        self.coefficient = self.value ** self.power
         self.__class__ = Variable
         self.power = [1]
         self.value = [intwrt]
 
+    def __radd__(self, other):
+        return self + other
+
+    def __add__(self, other):
+        if isinstance(other, Constant):
+            if other.power == self.power:
+                if self.before == '-':
+                    self.value = self.calculate() - other.calculate()
```

Comment:
Possibly do it the same way as you did in `__mul__`
```suggestion
                    result = Constant(self.calculate() - other.calculate())
```